pkg/liveness/liveness.go

```diff
@@ -58,13 +59,14 @@ func getLiveness(timeout time.Duration, csiConn *grpc.ClientConn) {
 }
 
 func recordLiveness(endpoint string, pollTime, timeout time.Duration) {
+	liveMetricsManager := metrics.NewCSIMetricsManager("")
```

Comment:
instead of `empty string use `conf.DriverName` you can refer to https://github.com/ceph/ceph-csi/pull/924/files#diff-2747365fb461f00ac376b491c6c15ea4R68

Reply:
Done.